extensions/notebook/src/extension.ts

```diff
@@ -113,6 +128,11 @@ export async function activate(extensionContext: vscode.ExtensionContext): Promi
 	extensionContext.subscriptions.push(vscode.window.registerTreeDataProvider(BOOKS_VIEWID, bookTreeViewProvider));
 	extensionContext.subscriptions.push(vscode.window.registerTreeDataProvider(READONLY_BOOKS_VIEWID, untitledBookTreeViewProvider));
 
+	extensionContext.subscriptions.push(vscode.commands.registerCommand('notebook.command.openCreatedBook', (bookPath: string) => {
```

Comment:
Why are we splitting this creation logic between a notebook and the extension? Wouldn't it be better to have all the logic for creating a valid Jupyter book be in one place? And then you'd just call openBook after the book has been created.

Reply:
between the opening the create book and 'openCreatedBook' we need to run jupyter book create command (!jupyter-book create "$book_name" --content-folder "$content_folder"). And that happens when the user runs the book. After which we have the book created on file system, we update the files and open the book.

Reply:
Yes, but why would you not have the notebook do all the work? Keeping everything in one place to create a book seems like a useful thing.